test/app/adBlockTest.js

```diff
@@ -0,0 +1,27 @@
+/* global describe, it, before */
+
+const Brave = require('../lib/brave')
+const selectors = require('../lib/selectors')
+const messages = require('../../js/constants/messages')
+
+describe('adBlock', function () {
+  function * setup (client) {
+    yield client
+      .waitUntilWindowLoaded()
+      .waitForVisible('#window')
+      .waitForVisible(selectors.urlInput)
+  }
+
+  describe('blocks ruled ads', function () {
+    Brave.beforeAll(this)
+    before(function *() {
+      yield setup(this.app.client)
+    })
+
+    it('blocks a known rule on a local url', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.newsbtc.com/2016/02/14/bitcoin-price-weekly-analysis-primed-for-lift-off-2/')
+        .isExisting('.adfixedLeft').should.eventually.be.false
```

Comment:
I don't think this is working, and also no remote URLs.

So this should be a html file in the test/fixtures directory and it should contain maybe a script, image, or iframe with a parameter that will get blocked, like:

?ad_box_=1

I'm not sure how to detect that the resource got blocked offf hand though.